eng/tsp-core/pipelines/jobs/website.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - template: /eng/tsp-core/pipelines/templates/install.yml
+
+  - displayName: Install browsers
+    script: pnpm exec playwright install
+
+  - name: Build
+    script: pnpm --filter "@typespec/website..." run build
```

Comment:
Ah, this is failing CI because `script` needs to be the first value in the mapping.

> Also, unlike standard YAML, Azure Pipelines depends on seeing stage, job, task, or a task shortcut like script as the first key in a mapping.

(https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/?view=azure-pipelines#see-also)

Reply:
of course because it would be too hard to respect a standard for microsoft